.github/workflows/intermediate-docker.yml

```diff
@@ -4,192 +4,56 @@ on:
   workflow_dispatch:
     inputs:
       GO_VERSION:
-        description: 'Go version'
+        description: "Go version"
         required: true
-        type: string
-        default: '1.21'
+        type: choice
+        options:
+          - "1.20"
```

Comment:
why remain 1.20

Reply:
we can change this to `1.21` as the default one